recipes/iir1/all/conanfile.py

```diff
@@ -50,6 +52,8 @@ def export_sources(self):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
+        if self.version == "1.9.0":
```

Comment:
Probably this should be 
```suggestion
        if tools.Version(self.version) < "1.9.1":
```

Reply:
Yes! This is what I wanted to ask in my previous comment, as using self.version is clearly a string comparison I was curious how to do this correctly for multiple versions. tools.Version is the answer :)